manifests/w/Wesang/Yogiyo/4.0.6/Wesang.Yogiyo.locale.en-US.yaml

```
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.6.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Wesang.Yogiyo
+PackageVersion: 4.0.6
+PackageLocale: en-US
```

```suggestion
PackageLocale: ko-KR
```

Graphical interface of this install wizard (and also main application after installation) is in Korean. All related URLs with `yogiyo.co.kr` are in Korean too.

Also this file should be renamed to `locale.ko-KR.yaml`.